docker/add_gitbase.py

```diff
@@ -17,5 +18,32 @@ def get_or_create_gitbase_db():
     db.session.add(dbobj)
     db.session.commit()
 
+    return dbobj
+
+
+def add_gitbase_tables():
+    dbobj = get_or_create_gitbase_db()
+    TBL = ConnectorRegistry.sources['table']
+    for table in dbobj.all_table_names_in_schema('gitbase'):
+        # table_name should match the one in the datasource for fetch_metadata to work
+        if db.session.query(TBL).filter_by(table_name=table).first():
+            continue
+        if db.session.query(TBL).filter_by(table_name='gitbase.' + table).first():
+            continue
+
+        # create table with original name and fetch columns
+        tbl = TBL(table_name=table)
+        tbl.database = dbobj
+        db.session.add(tbl)
+        db.session.commit()
+        tbl.fetch_metadata()
+
+        # rename with prefix and set source
+        tbl.table_name = 'gitbase.' + table
+        tbl.sql = 'select * from ' + table
```

Comment:
Does this create a sort of view of the table? For example, can I already hardcode a `WHERE` or `LIMIT` clause?

Reply:
yes. In supersets "views" are called "tables".
You can add WHERE, LIMIT, custom columns (for example `LANGUAGE(filename, content)`) and whatever else.